numpy/distutils/intelccompiler.py

```diff
@@ -2,7 +2,7 @@
 
 from distutils.unixccompiler import UnixCCompiler
 from numpy.distutils.exec_command import find_executable
-from numpy.distutils.msvc9compiler import MSVCCompiler
+from distutils.msvc9compiler import MSVCCompiler
```

Comment:
Not sure removing the `numpy` here makes any difference, but I suspect we might want `.distutils.msvc9compiler` if it is removed I can't test that, however.